### PR TITLE
chore: Add pre release workflow

### PR DESCRIPTION
**Pull request by ebebbington**
**Description**

* Adds a pre release workflow, which automates the process of us preparing for a new release (bump versions), and will carry out all the tasks we would usually do, for example updating `egg.json`, then the readme
    * If this PR is approved, i'll make pr's for the other repos
    * Simple as, when we are wanting to release,  just create a new branch called `release-v<MAJOR>.<MINOR>.<PATCH>`, and wait for the pull request, then once that PR is approved and merged, we can release the draft
* Uses `BumperService`

### CODE REVIEW

**Comment by ebebbington** [comment]
I'll update *secret* with the diagram if this PR is. merged, i believe the process/docs will be even simpler but still mention everything needed

File: console/bump_versions.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const branch: string = Deno.args[0].split("=")[1]; // ["--version", "release-vX.X.X"]
```
Comment (crookse) [inline console/bump_versions.ts:1]: this will need to change to use the `BumperService`

File: console/update_deno_version_strings.ts [hunk 2]
```diff
@@ -39,3 +39,16 @@ await Deno.writeFile(
   "./.github/workflows/bumper.yml",
```
Comment (crookse) [inline console/update_deno_version_strings.ts:39]: this will need to change to use the `BumperService`

File: .github/workflows/pre_release.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  # Make a PR to master from a new branch with changes, and delete the created one
+  update-versions:
+
+    # Only run when a release-v* branch is created, and not by drashbot
+    if: contains(github.ref, 'release-v') && !contains(github.event.sender.login, 'drashbot')
+
+    strategy:
+      matrix:
+        deno: ["1.4.2"]
```
Comment (crookse) [inline .github/workflows/pre_release.yml:16]: change to 1.4.6

Comment (crookse) [reply .github/workflows/pre_release.yml]: df26878eedbc0e5fbbcdf2d6

**Review by crookse — APPROVED** [review]
LGTM. nice work dude!

**Comment by ebebbington** [comment]
@crookse you happy for this to be merged? With all the changes made to services repo

**Review by crookse — APPROVED** [review]

**Pull request merged**